monkey/monkey_island/cc/resources/aws_exporter.py

```diff
@@ -0,0 +1,607 @@
+
+            import_response = securityhub.batch_import_findings(Findings=findings_list)
+            if import_response['ResponseMetadata']['HTTPStatusCode'] == 200:
+                return True
+            else:
+                return False
+        except UnknownServiceError as e:
+            logger.warning('AWS exporter called but AWS-CLI securityhub service is not installed')
+            return False
+        except Exception as e:
+            logger.exception('AWS security hub findings failed to send.')
+            return False
+
+    @staticmethod
+    def _handle_tunnel_issue(issue, instance_arn):
+        finding = \
+            {"Severity": {
+                "Product": 5,
+                "Normalized": 100
+            }, "RecordState": "ACTIVE",
+                "Title": "Weak segmentation - Machines were able to communicate over unused ports.",
+                "Description": "Use micro-segmentation policies to disable communication other than the required.",
+                "Remediation": {
+                    "Recommendation": {
+                        "Text": "Machines are not locked down at port level. Network tunnel was set up from {0} to {1}"
+                            .format(issue['machine'], issue['dest'])
+                    }
+                }}
+
+        if 'aws_instance_id' in issue:
+            finding["Resources"] = [{
+                "Type": "AwsEc2Instance",
+                "Id": instance_arn.format(instance_id=issue['aws_instance_id'])
+            }]
+        else:
+            finding["Resources"] = [{'Type': 'Other'}]
+
+        return finding
+
+    @staticmethod
+    def _handle_sambacry_issue(issue, instance_arn):
+        finding = \
+            {"Severity": {
+                "Product": 10,
+                "Normalized": 100
+            }, "RecordState": "ACTIVE", "Title": "Samba servers are vulnerable to 'SambaCry'",
+                "Description": "Change {0} password to a complex one-use password that is not shared with other computers on the network. Update your Samba server to 4.4.14 and up, 4.5.10 and up, or 4.6.4 and up." \
+                .format(issue['username']), "Remediation": {
+                "Recommendation": {
+                    "Text": "The machine {0} ({1}) is vulnerable to a SambaCry attack. The Monkey authenticated over the SMB protocol with user {2} and its password, and used the SambaCry vulnerability.".format(
+                        issue['machine'], issue['ip_address'], issue['username'])
+                }
+            }}
+
+        if 'aws_instance_id' in issue:
+            finding["Resources"] = [{
+                "Type": "AwsEc2Instance",
+                "Id": instance_arn.format(instance_id=issue['aws_instance_id'])
+            }]
+        else:
+            finding["Resources"] = [{'Type': 'Other'}]
+
+        return finding
+
+    @staticmethod
+    def _handle_smb_pth_issue(issue, instance_arn):
+        finding = \
+            {"Severity": {
```

Comment:
The whole finding structure is the same (with even some values being the same).
Create a function which receives product_severity, normalized_severity, record_state, title, description, recommendation, and returns the finding